docs/pages/functionality/management.md

```diff
@@ -10,8 +10,8 @@ list: include
 
 # Service Mesh Lifecycle Management
 
-Meshery supports a number of [different service meshes](installation/adapters), so that you can learn and compare different service mesh functionality. Once you have selected to operate a given service mesh, Meshery will assist you with ongoing activities.
-
+Meshery supports a number of [different service meshes](https://docs.meshery.io/service-meshes), so that you can learn and compare different service mesh functionality. Once you have selected to operate a given service mesh, Meshery will assist you with ongoing activities.
```

Comment:
Can we use a relative reference instead?

Reply:
Yes sure. Fixing that.